src/commands/login.rs

```diff
@@ -22,8 +73,33 @@ impl LoginSubcommand {
             }
         };
 
-        AuthStore::set_token(Some(&token))?;
+        AuthStore::set_token(Some(&token))?;*/
 
-        Ok(())
+        let client = reqwest::blocking::Client::new();
+        let device_code_response = client
+            .post("https://github.com/login/device/code")
+            .header("accept", "application/json")
+            .json(&serde_json::json!({
+                "client_id": "7bd503594a0f9a9f7ed3",
+                "scope": "read:user",
+            }))
+            .send()?
+            .json::<DeviceCodeResponse>()?;
+
+        println!(
+            "Go to {} and enter the code {}",
+            device_code_response.verification_uri, device_code_response.user_code
+        );
```

Comment:
Let's stick the URL on its own line with some more padding so that it's more likely to be visible without line wrapping.

Reply:
I'm hoping to be able to detect if the user is capable of using a web browser and automatically redirecting them to it. Google does this with their gcloud cli somehow so I think we should be able to do it too. This would make the login incredibly smooth if we can do it! Otherwise yeah this sounds good as well!

Reply:
We can use the `opener` crate to open the URL in their browser. I think we could prompt to see if they want to open the browser!